app/src/main/res/menu/menu_main.xml

```diff
@@ -1,16 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <item android:id="@+id/undo"
-          android:title="@string/menu_undo"
-          android:icon="@drawable/ic_action_undo"
-          android:showAsAction="always">
+<menu
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="schemas.android.com/tools" >
```

Comment:
Looks like there need's to be a `http://` in front of `schemas.android.com/tools` for` tools:ignore` to work.